pkgs/development/tools/electron/common.nix

```diff
@@ -182,7 +185,7 @@ in
     # build/args/all.gn
     is_electron_build = true;
     root_extra_deps = [ "//electron" ];
-    node_module_version = info.modules;
+    node_module_version = lib.toInt info.modules;
```

Comment:
I find it a bit funny that you opted to use `lib.toInt` here. But not in a discouraging way. Feel free to do whatever floats you boat. Also, I am no longer involved in maintaining electron.
And not wanting to touch the update script feels like a common pattern. Especially because this is returned as a string by the releases API.

Rather, because I find the chain of function invocations funny:

`builtins.fromJSON` reads it as string -> `lib.toInt` -> `if lib.isInt value` -> `toString` (instead of `mkGnString` if it would already be a string):

https://github.com/NixOS/nixpkgs/blob/71d3a69606c4600e7d6b59c0d5af2071d4f01f2c/pkgs/applications/networking/browsers/chromium/common.nix#L130-L151